public/app/features/logs/components/panel/LogListContext.tsx

```diff
@@ -237,6 +237,22 @@ export const LogListContextProvider = ({
   const [detailsWidth, setDetailsWidthState] = useState(getDetailsWidth(containerElement, logOptionsStorageKey));
   const [detailsMode, setDetailsMode] = useState<LogLineDetailsMode>(detailsModeProp ?? 'sidebar');
 
+  useEffect(() => {
+    reportInteraction('logs_log_list_logs_displayed', {
```

Comment:
not technically an interaction, but good data to have...
But regression in rendering this component could spam. Maybe global variable flag or something else to ensure that multiple re-renderings of this component in the same session won't fire again?

Reply:
Yeah session storage should work great

Reply:
Done that way.